module/os/windows/spl/spl-kstat.c

```diff
@@ -784,7 +784,7 @@ void
 kstat_set_string(char *dst, const char *src)
 {
 	memset(dst, 0, KSTAT_STRLEN);
-	(void) strncpy(dst, src, KSTAT_STRLEN - 1);
+	(void) strlcpy(dst, src, KSTAT_STRLEN);
```

Comment:
This fills the buffer with 0 and then fills it up excluding the very last char.
so buffer size is KSTAT_STRLEN
do we even need memset anymore?

Reply:
memset seems excessive